schema/event_mapping_entity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$udmi_version": "1.4.0",
+  "title": "Mapping Event Entity",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "code": {
+      "description": "",
+      "type": "string"
+    },
+    "type": {
+      "description": "",
+      "type": "string"
+    },
+    "cloud_device_id": {
+      "description": "",
+      "type": "string"
+    },
+    "connections": {
+    },
+    "links": {
+    },
+    "translation": {
+      "$ref": "file:building_config.json"
```

Comment:
the referenced thing isn't quite a building config if it's under a "translation" bucket, maybe it was off before but we should clean it up as we go along.  These look like "translations" (plural), and then the thing under it should have the word "translation(s)" in it.

Reply:
Plural translations to me implies multiple different takes rather than a singular work - Lattimore's Illiad is a translation (singular). 

Not sure what you're asking re: the clean up, do you just want me to rename the `building_config.json` file?